### DISCUSSION TITLE
Showing negative severity within cards/metrics

### DISCUSSION THREAD

**Discussion by JT1989** [twilio-labs/paste#952] [General] [answered]
Hi DSYS! 

Wanted to share feedback from the UX Review Crew:

_"The negative severity of messages in 'queue overflow' state doesn't come thru in the UI as much, and while link out to learn more about 'error 30001' is nice, more messaging/indication to the user that 'queue overflow' messages are in a bad state and what to do about them (right on the page) will help."_

For more context, they are referring to: 
<img width="600" alt="Screen Shot 2020-11-17 at 7 12 40 PM" src="https://user-images.githubusercontent.com/8630856/99478253-ee55a700-2908-11eb-9f2c-d3daed599aff.png">
<img width="600" alt="Screen Shot 2020-11-17 at 7 19 14 PM" src="https://user-images.githubusercontent.com/8630856/99478753-d2063a00-2909-11eb-9ef4-b05a7847d85d.png">

I'd like to follow-up with y'all:How might we create a pattern  to show metrics in a consistent way? In addition, how can we showcase the negative severity? 

For example, Product and Design folks have suggested a red background to address the UX Review crew's feedback:
<img width="225" alt="Screen Shot 2020-11-17 at 7 47 31 PM" src="https://user-images.githubusercontent.com/8630856/99480859-c9affe00-290d-11eb-9aa7-170bc16445be.png">

How might we have a shared language for metrics, specifically to call-out/emphasize severe, negative metrics?

I've taken the liberty of pulling together some screenshots to get the conversation started and so you can see it all together visually. 

**Messaging**
<img width="566" alt="Screen Shot 2020-11-17 at 7 12 48 PM" src="https://user-images.githubusercontent.com/8630856/99478320-0af1df00-2909-11eb-95c8-7aa2be0522a6.png">

**V&V**
<img width="1160" alt="Screen Shot 2020-11-17 at 7 23 02 PM" src="https://user-images.githubusercontent.com/8630856/99479147-78ead600-290a-11eb-809b-b6e935a72367.png">

**Phone Numbers** 
<img width="890" alt="Screen Shot 2020-11-17 at 7 23 44 PM" src="https://user-images.githubusercontent.com/8630856/99479156-7a1c0300-290a-11eb-848b-fa3f66e4103f.png">

There's likely other teams that I'm not aware of who may need to use this shared pattern.

**Comment by TheSisb** [comment, marked as answer]
Hey @JT1989, thanks for the detailed question!

I don't have a comprehensive answer, so I'm going to skip over some of the harder questions (i.e.: "How might we create a pattern to show metrics in a consistent way?") and leave some notes in this discussion for future readers.  

To a colorblind user, this image
![image](https://user-images.githubusercontent.com/1592327/99550335-9f4f5680-2988-11eb-9514-b71aeb73c9b8.png)
and this image 
![image](https://user-images.githubusercontent.com/1592327/99550368-ab3b1880-2988-11eb-8755-3093ccbf580f.png)
would look pretty much the same from a severity perspective, because adding more red coloring would not be parseable.

It likely ends up looking like this, which has no color, but has 'severity' to communicate:
![image](https://user-images.githubusercontent.com/1592327/99550524-ce65c800-2988-11eb-8acb-b39bdb4cdd17.png)


Therefore I wonder if this is less of a color prevalence problem, and rather an IA problem. I'm currently noodling with the thought that a different treatment of how the data is presented would be more clear and inclusive.  

That's all I have for now.

**Reply by JT1989** [reply]
Thanks @TheSisb , fwiw, I agree, I don't think coloring the background or changing the color text to red will make the severity any clearer, it seems to add noise for non-color blind users and it also doesn't successfully convey severity to color blind users. 

Excited to hear your ideas on how to best highlight information in an accessible way, once you have had a chance to noodle on it!
